src/sentry/lang/native/minidump.py

```diff
@@ -60,9 +60,27 @@ def process_minidump(minidump, cfi=None):
         return ProcessState.from_minidump_buffer(minidump, cfi)
 
 
+def write_minidump_dummy_data(data):
+    # Minidump events must be native platform.
+    data['platform'] = 'native'
+
+    # Create a placeholder exception. This signals normalization that this is an
+    # error event and also serves as a placeholder if processing of the minidump
+    # fails.
+    exception = {
+        'type': 'Minidump',
+        'mechanism': {
+            'type': 'minidump',
+            'handled': False,
+            'synthetic': True,
+        }
+    }
+    data['exception'] = {'values': [exception]}
+
+
 def merge_process_state_event(data, state, cfi=None):
     data['platform'] = 'native'
-    data['level'] = 'fatal' if state.crashed else 'info'
+    data.setdefault('level', 'fatal' if state.crashed else 'info')
```

Comment:
This, again, means that if `state.crashed is None`, you set `'info'`. In that case it should however **not** modify the `level` and just leave in what was set by ingestion (which should, in turn, default to `fatal`). In my opinion, this function here should not set the default.

Reply:
I now realize that this is `merge_process_state_event` (so, the code we want to delete), I just reverted the changes.